ansible_collections/arista/avd/roles/eos_designs/templates/network_services/main.j2

```diff
@@ -47,6 +48,12 @@
       switch.network_services_l3 is arista.avd.defined(true)) %}
 {%     include 'network_services/vtep-logic.j2' %}
 
+{%     if default_vrf.svi_subnets %}
```

Comment:
here you check for svi_subnets. Somewhere else you check for enabled. I don't think you need the enabled flag.

Reply:
Yeah, I made multiple changes. But I left that for testing. Will remove the enabled flag. I will modify the logic inside router_bgp. Instead of checking if vrf == default only I wil also check if there are any vlans under default vrf now. This will make sure the S-IRB with default vrf is configured only when there are vlans under default vrf.